_data/toc.yaml

```diff
@@ -1710,6 +1710,8 @@ manuals:
 
 - sectiontitle: Docker Hub
   section:
+  - path: /docker-hub/overview/
+    title: Overview
   - path: /docker-hub/
     title: Quickstart
```

Comment:
Wondering if we could still repurpose the `index.md` to be used as overview (and perhaps move `quickstart` to a `quickstart.md`, then

- `docs.docker.com/docker-hub/` would be the overview / landing page for hub (which is the most logical and shortest URL to look for)
- from there you could go to `docs.docker.com/docker-hub/quickstart/`

We can still call it `overview` in the TOC (we do that in most places), but effectively those overviews are the "index" pages for the section.

Reply:
Thanks @thaJeztah for the feedback. I can move the overview to index.md and quickstart to the new quickstart.md. I'll update it in another commit and tag you for review